Sources/Charcoal/Filters/Map/MapFilterViewController.swift

```diff
@@ -215,10 +215,7 @@ extension MapFilterViewController: MKMapViewDelegate {
     }
 
     func mapView(_ mapView: MKMapView, didUpdate userLocation: MKUserLocation) {
-        if hasRequestedLocationAuthorization {
```

Comment:
So we no longer need 'hasRequestedLocationAuthorization'?

Reply:
I don't think so. I can remove it

what do you think @vadymmarkov ?

Reply:
What's the difference between `centerOnInitialCoordinate` and `centerOnUserLocation`?

Reply:
on `centerOnInitialCoordinate` considers the last searched coordinate first, otherwise the user location, finally the default if all fail

`centerOnUserLocation` just considers the user location.

I chose `centerOnInitialCoordinate` because given the 2 possible sequences of calls, we don't want to move from a user search coordinate to the user position (because of timing)
so i use the same call both when the map loads and when the user location is ready

Reply:
I see. It makes sense to me. If `hasRequestedLocationAuthorization` is not used anywhere after your change I'm of course fine with removing it 😉